patchwork-events-entity/src/main/java/net/patchworkmc/impl/event/entity/EntityEvents.java

```diff
@@ -217,6 +219,19 @@ public static boolean onProjectileImpact(ThrownEntity throwable, HitResult ray)
 		return MinecraftForge.EVENT_BUS.post(new ProjectileImpactEvent.Throwable(throwable, ray));
 	}
 
+	public static boolean onTravelToDimension(Entity entity, DimensionType dimensionType) {
+		EntityTravelToDimensionEvent event = new EntityTravelToDimensionEvent(entity, dimensionType);
+		return !MinecraftForge.EVENT_BUS.post(event);
+		/* Forge has this thing:
```

Comment:
why comment this out?

Reply:
`StorageMinecart` doesn't seem to have `dropContentsWhenDead` - i assume it's a forge method

Reply:
It's the same behaviour as forge now - what do you mean by adding a // TODO? What should it say?